### ISSUE TITLE
Wrong parameters for the 4f elements

### ISSUE THREAD

**Issue by mretegan** [mretegan/crispy#31]
The truncated output from Cowan's RCN for Eu 4f7 electronic configuration:

```
   ETOT= -21658.42787 -21658.42787         -21663.30022 -21663.30022 -21663.10600 -21663.10600 -21663.10600


       4f7              5 -21663.1060 0   12.9967 1    8.1069 1    5.8191 1    0.1604 2



0    T(RUN)=     0.00,     T(JOB)=     0.00 SECONDS
```

Currently in Crispy: F2(4f,4f) = 8.721 compared to 12.9967 from above. The 5f elements are not affected.

**Comment by mretegan** [comment]
This is fixed in the latest release.

**Comment by mretegan** [comment]
Fixed in version 0.8.0.
